### PR TITLE
Remove images and use imgur to store them.

### PR DESCRIPTION
**Pull request by bogdandrutu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ While the Agent and Collector share the same binary, the configuration between
 the two may differ depending on requirements (e.g. queue size and feature-set
 enabled).
 
-![deployment-models](images/opentelemetry-service-deployment-models.png)
+![deployment-models](https://imgur.com/Tj384ap)
```
Comment (tigrannajaryan) [inline README.md]: Doesn't seem to be working for me.
This is what I see:
image

**Comment by bogdandrutu** [comment]
@tigrannajaryan done for the message. The squash will merge everything into one commit with the name of the PR.

File: README.md (hunk 1 above)
Comment (bogdandrutu) [reply README.md]: Done.

**Comment by bogdandrutu** [comment]
Hmm does anyone know why if I rebase and force push I loose all my aprovals?

**Comment by bogdandrutu** [comment]
My bad on this. When I rebase I missed the PR that I added via the web interface to fix the link to the image.

**Pull request merged**